setup.py

```diff
@@ -14,12 +13,12 @@
 
 
 # Check Python version.
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 6):
```

Comment:
If we are requiring >= 3.6 or >= 3.7, do we still need this check?

Reply:
Nope. Let me remove that.

Reply:
I removed the pip check too and let `pyproject.toml` handle the setup dependencies.